api/cedar_metadata_records/serializers.py

```diff
@@ -0,0 +1,116 @@
+import logging
+
+from django.db import IntegrityError
+from rest_framework import serializers as ser
+
+from api.base.exceptions import InvalidModelValueError, JSONAPIException
+from api.base.serializers import JSONAPISerializer, LinksField, RelationshipField
+from api.base.utils import absolute_reverse
+
+from osf.exceptions import ValidationError
+from osf.models import CedarMetadataRecord, CedarMetadataTemplate, Guid
+
+logger = logging.getLogger(__name__)
+
+
+class TargetRelationshipField(RelationshipField):
+
+    def get_object(self, _id):
+        return Guid.load(_id)
+
+    def to_internal_value(self, data):
+        return self.get_object(data)
+
+
+class CedarMetadataTemplateRelationshipField(RelationshipField):
+
+    def get_object(self, _id):
+        return CedarMetadataTemplate.load(_id)
+
+    def to_internal_value(self, data):
+        return self.get_object(data)
+
+
+class CedarMetadataRecordsSerializer(JSONAPISerializer):
+
+    class Meta:
+        type_ = 'cedar-metadata-records'
+
+    filterable_fields = frozenset(['is_published'])
+
+    id = ser.CharField(source='_id', read_only=True)
+
+    target = RelationshipField(
+        source='guid',
+        related_view='guids:guid-detail',
+        related_view_kwargs={'guids': '<guid._id>'},
```

Comment:
For CR: when a field in model and serializer differ, must use the model field name `guid` in `related_view_kwargs`, otherwise, target cannot be serialized. Ditto for `CedarMetadataRecordsCreateSerializer`.